imager/imager_images/models.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+# [('value', 'visible')]
+PUBLISH_CHOICES = (
+    ('public', 'Public'),
+    ('private', 'Private'),
+    ('shared', 'Shared'),
+)
+
+
+def user_file_store_path(instance, filename):
+    """File will be uploaded to MEDIA_ROOT/user_{id}/{filename}"""
+    return "user_{0}/{1}".format(instance.user.id, filename)
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    """Create photos for a user."""
+
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL,
+                              on_delete=models.CASCADE, related_name="photo")
+    title = models.CharField(max_length=30, blank=True)
+    description = models.CharField(max_length=60, blank=True)
+    file = models.ImageField(upload_to=user_file_store_path)
```

Comment:
You may want to rename this variable so that you're not overriding the Python `file` keyword - maybe something like `img_file`?